### DISCUSSION TITLE
ie11 devtools

### DISCUSSION THREAD

**Discussion by SergeyWebDeveloper** [react-hook-form/react-hook-form#4164] [Q&A]
I try to run the application in IE11, I get an error like this

![image](https://user-images.githubusercontent.com/12486958/107642746-54db2e80-6c86-11eb-8fe1-5474e019dfb2.png)

react-hook-form v.6.5.3
@hookform/devtools v.2.2.1

aliases are registered in webpack v.5
```
alias: {
      'react-hook-form': 'react-hook-form/dist/index.ie11',
    },
```
little-state-machine i don't use.

What am I doing wrong?

**Comment by SergeyWebDeveloper** [comment]
@bluebill1049, help please)

**Reply by bluebill1049** [reply]
try to include corejs, see if this issue gets resolved.

**Reply by SergeyWebDeveloper** [reply]
enabled, it doesn't help(

**Reply by SergeyWebDeveloper** [reply]
@bluebill1049, Can you run a @hookform/devtools for IE11?
